polkadot/node/core/pvf/execute-worker/src/lib.rs

```diff
@@ -157,90 +167,58 @@ pub fn worker_entrypoint(
 				let compiled_artifact_blob = match std::fs::read(&artifact_path) {
```

Comment:
Given that we are reading the artifact here already, could we restrict fs access for the child even further?

Reply:
Great point. Mind if we do it in a small follow-up? (https://github.com/paritytech/polkadot-sdk/issues/2164)

Reply:
Sure!